nexusadspy/client.py

```diff
@@ -114,8 +118,13 @@ def _do_paged_get(self, url, method, data=None, headers=None,
         return r_code, res
 
     def _do_throttled_request(self, url, method, data=None, headers=None,
-                              sec_sleep=2., max_failures=100, get_field=None):
-        params = self._get_params(method, data)
+                              params=None, sec_sleep=2., max_failures=100,
+                              get_field=None):
+        if params is not None:
```

Comment:
this shouldn't be necessary - all we want to do is pass separate optional `params` and `data` dictionaries all the way down to the one `requests` call in the code.